File: Service_eHospital_DongNai_A_NSTL/Implement/sysdiagramsService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class sysdiagramsService : BaseService<sysdiagrams, IsysdiagramsRepository>
    , IsysdiagramsService
    {
    private readonly IsysdiagramsRepository _sysdiagramsRepository;
    public sysdiagramsService(IsysdiagramsRepository sysdiagramsRepository) : base(sysdiagramsRepository)
    {
    _sysdiagramsRepository = sysdiagramsRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/sysdiagrams.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { sysdiagrams } from './sysdiagrams.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class sysdiagramsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'name', 'principal_id', 'diagram_id', 'version', 'definition'];

    List: sysdiagrams[] | undefined;
    ListFilter: sysdiagrams[] | undefined;
    FormData!: sysdiagrams;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "sysdiagrams";
    }
}

